static/js/StaticPages.jsx

```diff
@@ -2636,6 +2636,222 @@ const StaticHR = () =>
+            </div>
+        );
+    };
+    
+    // Show the list of job postings within a department category
+    const JobsListForDepartment = ({ jobsList }) => {
+        return (
+            <section className="jobsListForDepartment">
+                {jobsList.map((job) => (
+                    <Job key={job.id} job={job} />
+                ))}
+            </section>
+        );
+    };
+    
+    // Job postings are grouped by department. This component will show the jobs for a specific department
+    // Each department has a header for its category before showing a list of the job postings there
+    const JobPostingsByDepartment = ({ department, departmentJobPostings }) => {
+        return (
+            <section className="section department englishOnly">
+                <header>
+                    <h2 className="anchorable">{department}</h2>
+                </header>
+                <JobsListForDepartment key={department} jobsList={departmentJobPostings} />
+            </section>
+        );
+    };
+    
+    // Show all the job postings, but first group them by department and render each department separately
+    const JobPostings = ({ jobPostings }) => {
+        const groupedJobPostings = jobPostings.reduce((jobPostingsGroupedByDepartment, jobPosting) => {
+            const category = jobPosting.jobDepartmentCategory;
+            if (!jobPostingsGroupedByDepartment[category]) {
+                jobPostingsGroupedByDepartment[category] = [];
+            }
+            jobPostingsGroupedByDepartment[category].push(jobPosting);
+            return jobPostingsGroupedByDepartment;
+        }, {});
+    
+        return (
+            Object.entries(groupedJobPostings).map(([department, departmentJobPostings]) => {
+                return (
+                    <JobPostingsByDepartment
+                        key={department}
+                        department={department}
+                        departmentJobPostings={departmentJobPostings}
+                    />
+                );
+            })
+        );
+    };
+    
+    
+    const NoJobsNotice = () => {
+        return (
+            <div className="section nothing">
+                <p>
+                    <span className="int-en">
+                        Sefaria does not currently have any open positions.
+                        Please follow us on <a target="_blank" href="http://www.facebook.com/sefaria.org" >Facebook</a>
+                        to hear about our next openings.
+                    </span>
+                    <span className="int-he">
+                        ספריא איננה מחפשת כעת עובדים חדשים.
+                        עקבו אחרינו ב<a target="_blank" href="http://www.facebook.com/sefaria.org" >פייסבוק</a>&nbsp;
+                        כדי להשאר מעודכנים במשרות עתידיות.
+                    </span>
+                </p>
+            </div>
+        );
+    };
+
+    const [jobPostings, setJobPostings] = useState([]);
```

Comment:
The state and hooks are generally defined at the top of the component, for readability.